aad/spring-security/servlet/oauth2/client-access-multiple-resource-server/resource-server-1/pom.xml

```diff
@@ -11,7 +11,7 @@
     <relativePath>../../../../../../pom.xml</relativePath>
   </parent>
 
-  <artifactId>servlet-oauth2-client-access-multiple-resource-server-resourc-server-1-application</artifactId>
+  <artifactId>servlet-oauth2-client-access-multiple-resource-server-resource-server-1-application</artifactId>
```

Comment:
If you change this(from `resourc-` to `resource-`) here in this PR, do you think we should change all of them?
![image](https://user-images.githubusercontent.com/4465723/148973323-d2c8cb92-07ed-4824-bc23-2383d6855537.png)

Reply:
Updated.